source/how-we-work/version-control/support-branches.html.md

```diff
@@ -0,0 +1,111 @@
+
+![Graph showing the main and support branch running in parallel. The support/4.x branch diverges from the last v4 tag on the main branch. After the point the two branches diverge, the main branch has tags for v5.0.0 and v5.0.1 and the support branch has tags for v4.6.1 and v4.6.2.](/images/support-branch-prev-major.svg)
+
+<!-- 
+gitGraph
+    commit
+    commit tag: "v4.6.0"
+    branch "support/4.x"
+    checkout "main"
+    commit
+    commit
+    commit
+    commit tag: "v5.0.0"
+    commit
+    checkout "support/4.x"
+    commit
+    commit tag: "v4.6.1"
+    checkout "main"
+    commit
+    commit
+    commit tag: "v5.0.1"
+    checkout "support/4.x"
+    commit
+    commit tag: "v4.6.2"
+-->
+
+There should only be one support branch for each previous major version. The support branch should be named using only the major version number, for example the branch name for the v4 major release would be called `support/4.x`.
+
+Because the support branch will never be merged back into the main branch, any changes that affect both current and previous major versions require multiple pull requests -- one for each support branch for each of the affected previous major versions, and one to fix it on the main branch for the current major version.
+
+## Using support branches for hotfixes
+
+If we discover a bug in GOV.UK Frontend that we urgently need to fix, we can use a support branch to release just the fix without including other unreleased changes on main.
+
+(Ideally the main branch is always kept in a releasable state, but we might still prefer to release a set of changes together to provide a more coherent release, or hold off releasing a change until we have accompanying documentation ready to go.)
+
+Support branches should only be used to fix bugs, not to introduce new features.
+
+![Graph showing the main and support branch running in parallel. The support/4.6.x branch diverges from the v4.6.0 on the main branch. After the point the two branches diverge, the main branch has unreleased changes and the support branch has a tag for v4.6.1](/images/support-branch-hotfix.svg)
+
+<!-- 
+gitGraph
+    commit
+    commit tag: "v4.6.0"
+    branch "support/4.6.x"
+    checkout "main"
+    commit
+    commit
+    commit
+    checkout "support/4.6.x"
+    commit
+    commit tag: "v4.6.1"
+-->
+
+The support branch should be named using the major and minor version number of the previous release. For example, if the last release was v4.6.0 and we expect the unreleased changes on the main branch to go out as part of v4.7.0, the support branch should be named `support/4.6.x`.
+
+Because the support branch will never be merged back into the main branch, you may need to raise a second pull request to make the same changes to the main branch, otherwise the bug may be reintroduced in the next release.
```

Comment:
I reckon we should be a bit more assertive here. I can't think of a scenario where we wouldn't want changes in a support branch to go into main:

```suggestion
Because the support branch will never be merged back into the main branch, you will need to raise a second pull request to make the same changes to the main branch, otherwise the bug may be reintroduced in the next release.
```

Reply:
I think there are a few cases where you might not need to raise the PR against main. For example, we may have fixed the bug already on `main` and later decided it was worth releasing from a support branch because there are changes on `main` that aren't ready to go out.

Reply:
That's true. Do we want to be clearer about that here? Something like:
```suggestion
Because the support branch will never be merged back into the main branch, you will need to raise a second pull request to make the same changes to the main branch if the change was introduced in the support branch but **not** the main branch, otherwise the bug may be reintroduced in the next release.
```

There's probably a better way to word that than what I've suggested. Maybe it's worth writing up the concept of backporting from main cause we think previous versions will benefit separately in the guidance? I couldn't find anything in there currently that touches on it.

Reply:
I've reworked this paragraph to front load the important bit ('Make sure that the bug is also fixed on the `main` branch, otherwise the bug will be introduced in the next release.') and split out the bit that _might_ not be needed ('This may mean you need to raise a second pull request with the same changes against `main`.')

Hopefully that covers it.